static/js/ducks/sources.js

```diff
@@ -12,6 +12,9 @@ const FETCH_PENDING_GROUP_SOURCES = "skyportal/FETCH_PENDING_GROUP_SOURCES";
 const FETCH_PENDING_GROUP_SOURCES_OK =
   "skyportal/FETCH_PENDING_GROUP_SOURCES_OK";
 
+const FETCH_FAVORITE_SOURCES = "skyportal/FETCH_FAVORITE_SOURCES";
+const FETCH_FAVORITE_SOURCES_OK = "skyportal/FETCH_FAVORITE_SOURCES_OK";
+
```

Comment:
I think you want a REFRESH_FAVORITE_SOURCES websocket message here as well, so that the favorites page is also updated upon changes (right now only the star icons are updated as those rely on the favorites ducks but not the source table on the favorites page)

Reply:
Yes but updating the table automatically could be sort of annoying, as the MUI table will have to re-render whenever something is added / removed. So I think maybe it is better to have the favorites table only refresh on a reload, like the group sources page...